### ISSUE TITLE
Problem with permission of Camera on Android 6.0

### ISSUE THREAD

**Issue by jiahaoliuliu** [Ziggeo/Android-Client-SDK#1]
Hi:
I am running this sample app on Nexus 9 with Android 6.0 and I cannot user the camera. The error I received is this:

```
11-04 14:40:12.261 E/CameraService(27581): Permission Denial: can't use the camera pid=32570, uid=10317
11-04 14:40:12.311 E/AndroidRuntime(32570): FATAL EXCEPTION: main
11-04 14:40:12.311 E/AndroidRuntime(32570): Process: com.ziggeo.ziggeodemowithoutsources, PID: 32570
11-04 14:40:12.311 E/AndroidRuntime(32570): java.lang.RuntimeException: Fail to connect to camera service
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at android.hardware.Camera.<init>(Camera.java:495)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at android.hardware.Camera.open(Camera.java:341)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at com.ziggeo.androidsdk.recording.CameraManager.openCamera(CameraManager.java:79)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at com.ziggeo.androidsdk.recording.VideoRecordingManager.surfaceCreated(VideoRecordingManager.java:66)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at android.view.SurfaceView.updateWindow(SurfaceView.java:582)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at android.view.SurfaceView$3.onPreDraw(SurfaceView.java:177)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at android.view.ViewTreeObserver.dispatchOnPreDraw(ViewTreeObserver.java:944)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2055)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1107)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6013)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:858)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at android.view.Choreographer.doCallbacks(Choreographer.java:670)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at android.view.Choreographer.doFrame(Choreographer.java:606)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:844)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at android.os.Handler.handleCallback(Handler.java:739)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at android.os.Handler.dispatchMessage(Handler.java:95)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at android.os.Looper.loop(Looper.java:148)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at android.app.ActivityThread.main(ActivityThread.java:5417)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at java.lang.reflect.Method.invoke(Native Method)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
11-04 14:40:12.311 E/AndroidRuntime(32570): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

Could you take a look on it?

**Comment by 3akat** [comment]
Hi @jiahaoliuliu 
You are using a real device or an emulator?

**Comment by jiahaoliuliu** [comment]
@3akat Real device

**Comment by 3akat** [comment]
@jiahaoliuliu 
fixed in version 0.1.2

**Comment by jiahaoliuliu** [comment]
Thanks. I will test it today
